src/Resources/config/services.xml

```diff
@@ -38,6 +38,7 @@
             <service id="maker.doctrine_helper" class="Symfony\Bundle\MakerBundle\Doctrine\DoctrineHelper">
                 <argument /> <!-- entity namespace -->
                 <argument type="service" id="doctrine" on-invalid="ignore" />
+                <argument key="$attributeMappingSupport">%maker.compatible_check.doctrine.supports_attributes%</argument>
```

Comment:
I get "Invalid constructor argument 4 for service "maker.doctrine_helper": argument 3 must be defined before. Check your service definition.".